pages/profiles/ggwilliams.md

```diff
@@ -0,0 +1,38 @@
+# Gabrielle (Gabby) Williams
+
+Eastern Standard Time
+
+Mac OS Sierra
+
+My name is *Gabby* I just finished the General Assembly WDI program in **Boston** where I learned the following languages:
+1. Javascript
+2. Ruby on Rails
+3. Ember
+4. and more...
+
+I hope to learn a lot more through [Open Learning Exchange](https://github.com/open-learning-exchange/open-learning-exchange.github.io "Open Learning Exchange")
+This is my project portfolio site [portfolio](https://ggwilliams.github.io/portfolio//)
+
+I enjoy:
+* FrontEnd 
+* Javascript Algorithms
+* Code Challenges
+
+When I have ~~freetime~~, I like to:
```

Comment:
When you read it without the cross out it reads "When i have, i like to:". Could change it to "When i have ~~free~~time, i like to:"